skimage/segmentation/_join.py

```diff
@@ -114,20 +116,29 @@ def relabel_sequential(label_field, offset=1):
     >>> relab
     array([5, 5, 6, 6, 7, 9, 8])
     """
+    offset = int(offset)
+    if offset <= 0:
+        raise ValueError("Offset must be strictly positive.")
+    if int(label_field.min()) < 0:
```

Comment:
The challenge with these kinds of checks is that they are expensive. What was the error message without that check?

Reply:
I guess max is used below. so this isn't really that big of an increase in the cost.. Generally I'm against these kinds of checks.

Reply:
I generally agree, but `relabel_sequential` is not a function you'd typically call in a hot loop.